distribution/docker/solr-cloud/src/main/resources/docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+version: '3.5'
+services:
+  zk1:
+    image: zookeeper:${docker.zookeeper.version}
+    restart: always
+    ports:
+    - "${docker.zookeeper.port.1}:2181"
+    - "${docker.zookeeper.port.2}:2888"
+    - "${docker.zookeeper.port.3}:3888"
+    environment:
+      ZOO_MY_ID: 1
+      ZOO_SERVERS: server.1=zk1:2888:3888
+      JVMFLAGS: "-Xms48m -Xmx128m"
+    hostname: zk1
+  solr1:
+    image: codice/ddf-solr:${project.version}
+    restart: always
+    ports:
+    - ${docker.solr.port.1}:${docker.solr.port.1}
+    command: ["/opt/solr/bin/solr",
+              "start",
+              "-f",
+              "-cloud",
+              "-m", "1500m",
+              "-z", "zk1:2181",
+              "-p", "${docker.solr.port.1}", "-h", "solr1"]
+    hostname: solr1
+    volumes:
+    - type: bind
+      source: ${docker.solr.backup.directory.local}
```

Comment:
❓ Just curious, is there a reason we are bind mounting the backup instead of using a named volume?

Reply:
this docker compose is meant for running a solr cloud instance locally for development purpose. we wouldn't what volume to mount to.

Reply:
I was thinking we could use a named volume like `solr-backup` that would be initially empty and gets populated during development. If the containers are stopped and restarted, the same data would be available as long as the volume isn't deleted. If the volume is deleted, then a new empty `solr-backup` volume would be created.

Reply:
ah ye, i misread. good idea creating a docker volume.